nixos/modules/services/databases/memcached.nix

```diff
@@ -16,45 +16,46 @@ in
 
   options = {
 
-    services.memcached = {
+    services.memcached = with types; {
```

Comment:
(nitpick, probably irrelevant) I'm not sure if we actually benefit from adding even more `with ...;` expressions. While I think that those are pretty handy and convenient, I know that especially beginners will have a hard time understanding where a variable is *actually* coming from if it was passed through several expressions using `with ...;`.